modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/emails_list/EmailsList.js

```diff
@@ -153,7 +153,13 @@ function EmailsList({dataLoading, dataSetContext, items}) {
 	}
 
 	if (!items?.length) {
-		return <EmptyResultMessage />;
+		return (
+			<ClayEmptyState
+				description={Liferay.Language.get('no-items-were-found')}
+				imgSrc="/o/classic-theme/images/states/empty_state.gif"
+				title=""
```

Comment:
Please apply consistent `ClayEmptyState` props on all usages in this PR:
- description: there-are-no-results
- imgSrc: use `themeDisplay.getPathThemeImages`
- title: no-results-found.

I don't have a strong opinion on what messages should be in description and title. I'd only like them to be as consistent as possible. Looking at [Clay defaults](https://github.com/liferay/clay/blob/master/packages/clay-empty-state/src/index.tsx), they are hardcoded to English, so it's better to set them localized in usages. I have a very minor gripe with `items` in `no-items-were-found`, since it would look a bit awkward in something like a table of users :smile:

Reply:
Oppps my bad here! I focused on DataSet.js and forgot the rest LOL. Thxs for pointing it out.

I'll set the description to a localized value 👍

Reply:
> they are hardcoded to English, so it's better to set them localized in usages
Indeed, thxs
![image](https://user-images.githubusercontent.com/19485114/161063431-88b90f53-f1fa-4ab6-867a-7a32d2bd4d74.png)

Reply:
> they are hardcoded to English, so it's better to set them localized in usages

https://github.com/liferay/clay/issues/4784